docs/docs/how_to/semantic-chunker.ipynb

```diff
@@ -297,13 +297,85 @@
     "print(len(docs))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Gradient\n",
+    "\n",
+    "In this method, the gradient of distance is used to split chunks along with the percentile method.\n",
+    "This method is useful when chunks are highly correlated with each other or specific to a domain e.g. legal or medical. The idea is to apply anomaly detection on gradient array so that the distribution become wider and easy to identify boundaries in highly semantic data."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "423c6e099e94ca69"
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "id": "b1f65472",
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "text_splitter = SemanticChunker(\n",
+    "    OpenAIEmbeddings(), breakpoint_threshold_type=\"gradient\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Madam Speaker, Madam Vice President, our First Lady and Second Gentleman.\n"
+     ]
+    }
+   ],
+   "source": [
+    "docs = text_splitter.create_documents([state_of_the_union])\n",
+    "print(docs[0].page_content)"
+   ],
+   "metadata": {},
+   "id": "e9f393d316ce1f6c"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "26\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(len(docs))"
+   ],
+   "metadata": {},
+   "id": "a407cd57f02a0db4"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "outputs": [],
+   "source": [],
+   "metadata": {},
+   "id": "3b45b2b24672590f"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "256b9e486f54663d"
```

Comment:
Can you delete the empty code boxes at the bottom of the doc

Reply:
Sorry, Missed that.
Deleted!